app/style/SendPage.less

```diff
@@ -130,6 +130,8 @@
 .send-address-wallet-icon {
   .inline-block-mixin;
 
+  margin-left: 0.5em;
```

Comment:
This made the +/- buttons get unaligned to the top wallet accounts on my screen. You should decrease the .send-input-form width to 82% to get them to align again.

Reply:
Oversight, sorry about that.